docs/workflows/lazy-downloading.rst

```diff
@@ -0,0 +1,60 @@
+Lazy Downloading
+================
+
+Pulp can sync content in a few modes: 'immediate', 'on_demand', and 'streamed'. Each provides a
+different behavior on how and when Pulp acquires content. These are set as the `policy` attribute
+of the `Remote` performing the sync. Policy is an optional parameter and defaults to `immediate`.
+
+immediate
+  When performing the sync, download all Artifacts now. Also download all metadata now to create
+  the content units in Pulp, associated with the RepositoryVersion created by the sync. `immediate`
+  is the default.
+
+on_demand
+  When performing the sync, do not download any Artifacts now. Download all metadata now to create
+  the content units in Pulp, associated with the RepositoryVersion created by the sync. Clients
+  requesting content trigger the downloading of Artifacts, which are saved into Pulp to be served to
+  future clients.
+
+  This mode is ideal for saving disk space because Pulp never downloads and stores packages that
+  clients don't need.
+
+streamed
+  When performing the sync, do not download any Artifacts now. Download all metadata now to create
```

Comment:
Could you add links to the glossary for anything in there? Artifacts, RepositoryVersion, etc

    :term:`artifacts<artifact>`

Reply:
yes fixed in next push.

Reply:
this came out really nicely I think.